src/components/Experiment/QuickOutput/InputPreview.js

```diff
@@ -5,19 +5,27 @@ import useBEMNaming from "../../../common/useBEMNaming";
 const defaultProps = {
   className: "input-preview",
   input: "",
-  onBackClicked: () => {},
+  onBackClicked: () => { },
   inputType: "image", // TODO: Change this default?
 };
 
 export default function InputPreview(givenProps) {
   const props = { ...defaultProps, ...givenProps };
   const { getBlock, getElement } = useBEMNaming(props.className);
 
+  const inputTypes = {
+    image: "Image",
+    audio: "Audio",
+    text: "Text",
+  };
+
+
   const getInput = () => {
     switch (props.inputType) {
       case "text":
         return <p className={getElement("text")}>{props.input}</p>;
-      case "audio":  // Currently not being used
+      case "audio":
+        return <audio className={getElement("audio")} controls src={props.input} />;
       case "image":
       default:
```

Comment:
Can you actually update this so that `case "image":` returns the `<img>` tag, and `default` returns a generic `<p>` that says `Not currently supported`?